modules/openideal_footer/src/Plugin/Block/OpenidealFooterPoweredByBlock.php

```diff
@@ -57,15 +63,44 @@ public static function create(ContainerInterface $container, array $configuratio
     );
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form = parent::blockForm($form, $form_state);
+    $config = $this->getConfiguration();
+
+    $form['openideal_official_site'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Official site'),
+      '#description' => $this->t('The OpenideaL official site.'),
+      '#default_value' => $config['openideal_official_site'] ?? self::OPENIDEAL_OFFICIAL_SITE,
```

Comment:
I'm just wondering why this is a part of the block configuration instead of `openideal_footer.openideal_footer_links_config`.

Doesn't it make sense to keep all footer-related configuration in one place?

Reply:
Yes. 
Done.